src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -1100,43 +1118,49 @@ public boolean isResourcePackRequired()
 	@Override
 	public boolean hasWhitelist()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.isWhitelistEnabled;
 	}
 
 	@Override
 	public void setWhitelist(boolean value)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		this.isWhitelistEnabled = value;
 	}
 
 	@Override
 	public boolean isWhitelistEnforced()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.isWhitelistEnforced;
 	}
 
 	@Override
 	public void setWhitelistEnforced(boolean value)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		this.isWhitelistEnforced = value;
 	}
```

Comment:
Should non-whitelisted players be kicked if this (or maybe `#setWhitelist`) is set to false?